src/frontend/Komponenter/Journalføring/JournalføringPdfVisning.tsx

```diff
@@ -1,37 +1,34 @@
-import PdfVisning from '../../Felles/Pdf/PdfVisning';
 import React, { useEffect } from 'react';
 import { HentDokumentResponse } from '../../App/hooks/useHentDokument';
 import styled from 'styled-components';
-import { Button } from '@navikt/ds-react';
+import DataViewer from '../../Felles/DataViewer/DataViewer';
 
-const FlexKnapper = styled.div`
-    display: flex;
-    justify-content: space-between;
+const Container = styled.div`
+    flex: 1 1 auto;
 `;
 
 const JournalføringPdfVisning: React.FC<{ hentDokumentResponse: HentDokumentResponse }> = ({
     hentDokumentResponse,
 }) => {
-    const { hentFørsteDokument, hentForrigeDokument, hentNesteDokument, valgtDokument } =
-        hentDokumentResponse;
+    const { hentFørsteDokument, valgtDokument } = hentDokumentResponse;
```

Comment:
`useHentDokument` kan sikkert få en liten opprydding nå :-)
Er det sånn at man må klikke på hvert dokument nå for å vise det, og at man ikke har noen andre knappet for "neste" og "forrige"?

Reply:
Man må klikke på dokumentnavnet eller papirikonet på samme rad for å vise dokumentet, ja

Reply:
Til opplysning så får man opp en loader med 'systemet laster' mens man venter på å se dokumentet 👍

Reply:
Fint! Ah jag tyckte bara jag fick upp noe grått lokalt